edweiss-app/components/core/PeriodBlock.tsx

```diff
@@ -3,29 +3,32 @@ import TView from './containers/TView';
 import formatTime from './formatTime';
 import TText from './TText';
 
-export const PeriodBlock = ({ period, course, user }: {
-    period: CourseTimePeriod; course: { id: string; data: Course; }; user: any;
+export const PeriodBlock = ({ period, course, user, format }: {
+    period: CourseTimePeriod; course: { id: string; data: Course; }; user: any; format: string;
 }) => {
+    const direction = format == 'week' ? 'column' : 'row';
+    const size1 = format == 'week' ? 12 : 15;
+    const size2 = format == 'week' ? 9 : 12;
```

Comment:
You could reduce 'brain' complexity of this part. I also am pationated anout ternary operator, I thing that they are great. Just in this case you could do instead is define the defaault values like `const size1 = 15 ... all others` and then add below one single if like this one `if format == 'week' { ... other values }`, this way you have fail safe default, less complexer code and cleaner code. I know that a small detail, but hay I am here for this 😉.

Reply:
Great! Looking forward.